app/src/pages/inside/dashboardPage/widgets/components/testsTableWidget/pTypes.js

```diff
@@ -0,0 +1,52 @@
+import { string, node, shape, arrayOf, bool, number, oneOf } from 'prop-types';
```

Comment:
Please follow the project code style and describe propTypes directly in component file

Reply:
Removed propTypes that weren't used across multiple component